pynxtools/dataconverter/readers/ellips/reader.py

```diff
@@ -466,7 +468,10 @@ def read(
         )
         template["/ENTRY[entry]/definition/@version"] = get_nexus_version()
         template["/ENTRY[entry]/program_name"] = "pynxtools"
-        template["/ENTRY[entry]/program_name/@version"] = version("pynxtools")
+        try:
```

Comment:
what about pynxtools.get_version() which would always give back something sensible

Reply:
Yes, this would be preferred and I also thought to generalise it here. However, we wanted to write these metadata outside the specific readers anyways, because pynx already knows everything from its own settings and CLI arguments. That's why I kept this out of here for now but eventually the whole version handling should be moved out of the reader. #221 is the related issue.